cabal.project.freeze

```diff
@@ -166,7 +166,7 @@ constraints: any.Boolean ==0.2.4,
              any.gargoyle-postgresql-nix ==0.3.0.1,
              any.generically ==0.1.1,
              any.generics-sop ==0.5.1.3,
-             any.ghc-bignum ==1.2,
+             any.ghc-bignum >=1.2,
```

Comment:
This sort of defeats the purpose of a "freeze" file :) Besides, I'm a little puzzled. We're just removing dependencies in this PR, so why was it necessary to relax this constraint?

Reply:
I see your point. I went ahead and just created another one.